docs/platform/howto/create_new_service_user.md

```diff
@@ -16,7 +16,16 @@ You can add service users for all the Aiven services except Aiven for Apache Fli
 and Aiven for Grafana®.
 
 :::note
-By default, the maximum amount of users allowed on a service is 50.
+By default, the maximum amount of users allowed on a service is 50. This limitation doesn't
+apply to the following services:
```

Comment:
services -> Aiven services? more consistent with the wording above about Flink

Reply:
I mean: "This limitation does not apply to the following Aiven services"

Reply:
yeah, to be perfectly precise "Aiven-managed services"